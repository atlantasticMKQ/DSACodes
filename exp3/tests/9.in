{{{{{{},3,{}},5,{{},8,{}}},2,{{},4,{}}},9,{{{},7,{}},6,{{},1,{}}}},10,{}}
