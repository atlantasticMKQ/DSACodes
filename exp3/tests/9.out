10,9,2,5,3,8,4,6,7,1
10,9,2,6,5,4,7,1,3,8
4
4
5